### ISSUE TITLE
When getting a server misbehave error, there is no way to retry the request

### ISSUE THREAD

**Issue by mulderbaba** [kitematic/kitematic#1046]
Getting the error:
Error while pulling image: Get https://index.docker.io/v1/repositories/library/cassandra/images: dial tcp: lookup index.docker.io on 193.140.80.9:53: server misbehaving

but there is no way to retry the downloading phase. It just stuck that way as seen in the screenshot.

<img width="1378" alt="screen shot 2015-09-17 at 00 04 25" src="https://cloud.githubusercontent.com/assets/603285/9918465/ac7d08ba-5ccf-11e5-8c15-cc3f5cc28009.png">

**Comment by FrenchBen** [comment]
If you have in your preferences to 'shutdown Linux VM on closing Kitematic' checked, then you should be able to close kitematic (which shuts down the VM) and should also kill the above pending download.

**Comment by mulderbaba** [comment]
yup, I have it checked. the thing is, I need to retry on this again and again until the IP is not misbehaving. IMO there should be a way to handle it, a refresh button maybe?

**Comment by FrenchBen** [comment]
Do you meant the ability to retry the download over and over when it fails?

**Comment by mulderbaba** [comment]
yes.
